schemas/com.snowplowanalytics.snowplow/browser_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for browser contexts",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "browser_context",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "viewport": {
+      "type": "string",
+      "maxLength": 20,
+      "description": "Viewport dimensions of the browser. Arrives in the form of WidthxHeight e.g. 1200x900."
+    },
+    "documentSize": {
+      "type": "string",
+      "maxLength": 20,
+      "description": "Document dimensions. Arrives in the form of WidthxHeight e.g. 1200x900"
+    },
+    "resolution": {
+      "type": "string",
+      "maxLength": 20,
+      "description": "Device native resolution. Arrives in the form of WidthxHeight e.g. 1200x900"
+    },
+    "colorDepth": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 1000,
```

Comment:
If we bump this down to 128 we can take advantage of SMALLINT (2 bytes) in Redshift.

Reply:
I cannot think of a screen for a few years that will have 128-bit color depth, except if it is a browser bug. Considering this will be a highly used context, I think the tradeoff could be worth it.
Will change.

Reply:
Yeah, and if it's a bug in a browser I think we probably want to generate a failed event anyway.

Reply:
Re:
Checked our database column translation docs and seems to be fine since the value is less than `32767`.

![image](https://user-images.githubusercontent.com/15251081/224293845-bacf8999-0ce0-4040-aac1-87513289b50e.png)